docker/start_migrate

```diff
@@ -6,6 +6,6 @@ set -o nounset
 
 
 echo "Django migrate"
-python manage.py migrate --noinput
+python manage.py migrate_multi --noinput
```

Comment:
It's simpler to have `migrate_multi` than conditionally run `migrate  --database=secondary`